docs.md

```diff
@@ -243,7 +243,7 @@ def beSmallerThan(value: Int) = ValidationRule[Int](s"be smaller than $value") {
   
 val child: RequestReader[User] = 
   (RequiredParam("name") ~
-  RequiredIntParam("age").should(beNonNegative and beSmallerThan(18))) map {
+  RequiredParam("age").as[Int].should(beNonNegative and beSmallerThan(18))) map {
```

Comment:
Yes, `beLessThan` is also the proposed name for the predefined rule in #164. I'll change it.